src/containers/my-resources/questions-container/QuestionsContainer.constants.tsx

```diff
@@ -13,21 +12,20 @@ import {
   AdditionalPropsInterface
 } from '~/types'
 import { getFormattedDate } from '~/utils/helper-functions'
+import { CheckIcons } from '~/utils/check-icons'
 import { styles } from '~/containers/my-resources/questions-container/QuestionsContainer.styles'
 
 export const columns: TableColumn<Question>[] = [
   {
     label: 'myResourcesPage.questions.title',
     field: 'title',
     calculatedCellValue: (item: Question) => {
+      const icon = CheckIcons(item.type)
```

Comment:
maybe just pass CheckIcons(item.type) below into icon and remove {} brackets to return instanty

Reply:
done